### PR TITLE
Running test suite through Github Actions. 

### PR DESCRIPTION
**Pull request by adnanhemani**
[X] Wrote test for feature
[N/A] Added changes to CHANGELOG.md
[N/A] Bumped version number (delete if unneeded)

**Changes proposed:**
Continuation of #525. Some minor changes were required to the code in #525 and I've further added Coveralls integration back in as well.

### CODE REVIEW

**Comment by adnanhemani** [comment]
Seems like I don't have push access to the original PR branch :( 

So we've got two options now (whichever is faster):
1. An approved review approved this PR (and then I can merge quickly) - @davidwagner @papajohn @JinwooPark00 (I believe now has push access)
2. @preethamsura - if you want to pull my changes into your branch and re-open your PR, I can approve and merge it in.

I took a quick look at the other outstanding PRs and I'm uncomfortable merging/reviewing anything else without having our testing/Coveralls back in place with this PR. Let's aim to resolve this ASAP so we can unblock the remaining PRs :)

**Comment by adnanhemani** [comment]
Testing Github Actions can be found here: [https://github.com/adnanhemani/datascience/actions](https://github.com/adnanhemani/datascience/actions)

**Review by davidwagner — APPROVED** [review]

**Comment by davidwagner** [comment]
@adnanhemani Oh gosh sorry for being so slow to respond!  I haven't logged into Github in a long time.  I approved this PR and merged it.  What are the next steps?

**Pull request merged**
